src/app.js

```diff
@@ -47,11 +49,17 @@ fluid.defaults("gpii.app", {
                 args: ["{that}"]
             },
             createOnEvent: "onGPIIReady"
+        },
+        settingsWindow: {
+            expander: {
+                funcName: "gpii.app.makeSettingsWindow"
+            },
+            createOnEvent: "onGPIIReady"
```

Comment:
These definitions are a mistake in the original app - should work together with @michelled to talk about them. Members are always instantiated immediately on construction and can't be delayed by "createOnEvent" declarations

Reply:
This is fixed in the 2092 branch: https://github.com/GPII/gpii-app/pull/17

Reply:
Ok, @danailbd - could you merge up your branch with that one, since it is where the active work on the app is taking place - thanks. It will also get you some more recent dependencies

Reply:
@amb26, we were hoping to avoid merging "in progress" branches. It makes diffs larger (in case the merged branches stay unmerged) and the git history a bit of a mess after everything is merged into "master". What do you usually do in similar situations :?

Reply:
That's my ideal too - unfortunately, that branch is taking a lot longer to complete than I expected. One thing we could do is comment out the running of the integration tests in my branch and then merge it to the hst branch. Even without the integration tests, there are several code improvements and some refactoring that you would benefit from. Would you be comfortable with that  @amb26? 
This can only happen after https://github.com/GPII/windows/pull/144 is merged.

Reply:
@danailbd, @michelled  - https://github.com/GPII/windows/pull/144 is now in, which is one less impediment. Personally I have no problem with basing this work on the in progress branch, because it is absolutely certain that https://github.com/GPII/gpii-app/pull/17 will be merged, and we find that our git history is already pretty tangled anyway since we never make efforts towards those things that some projects do to tidy it up, e.g. commit squashing, rebasing etc. After everything has been merged, the resulting git history will be no different to one in which multiple contributors had alternated by merging their work to their own versions of pull branches which they had then repeatedly reopened as pulls, which is something we have seen many times.

In order to keep the diffs down during review, though, you should reopen this pull as one against #17, and just make a comment with a link back to this pull so we can remind ourselves about what review there has been so far.

I've got no problems merging #17 to the HST branch since I understood we had been doing this regularly anyway (assuming there is nothing in it that the pilots team find objectionable). We just need to maintain the standard that things are not merged to overall master without sufficient tests.

Reply:
Once GPII/windows#144 lands in the hst branch for windows, I'll update my branch and we can push it.